modules/rosetta/src/main/scala/org/tessellation/rosetta/server/model/dag/schema.scala

```diff
@@ -11,20 +11,43 @@ object schema {
 
   case class GenericMetadata()
 
+  @derive(eqv, show)
+  sealed trait SignatureType extends EnumEntry
+
+  object SignatureType extends Enum[SignatureType] with SignatureTypeEncoder {
+    val values = findValues
+
+    case object Ecdsa extends SignatureType
+    case object EcdsaRecovery extends SignatureType
+    case object Ed25519 extends SignatureType
+    case object Schnorr1 extends SignatureType
```

Comment:
Are these values required? I'm almost sure they are not used in our case (we use Ecdsa AFAIR).

Reply:
They are defined by the [Rosetta docs](https://www.rosetta-api.org/docs/models/SignatureType.html), so I am adding them just in case.